tests/ProxyManagerTest/Generator/MethodGeneratorTest.php

```diff
@@ -162,7 +162,7 @@ public function testReturnTypeHintGeneration(string $methodName, string $expecte
     /**
      * @return string[][]
      */
-    public function returnTypeHintsProvider() : array
+    public static function returnTypeHintsProvider() : array
```

Comment:
Interesting change: is this now something endorsed in PHPUnit? It makes sense, since providers should not rely on test state...

Reply:
Honestly, I couldn't find anything related to that, but it makes more sense to me as well.